rma/wizards/rma_make_picking_view.xml

```diff
@@ -134,4 +134,27 @@
         </field>
     </record>
 
+    <record id="view_rma_button_form" model="ir.ui.view">
+        <field name="name">rma.order.form</field>
+        <field name="model">rma.order</field>
+        <field name="inherit_id" ref="rma.view_rma_form" />
+        <field name="arch" type="xml">
+            <header position="inside">
+                <button
+                    name="%(action_rma_picking_in)d"
+                    string="Create Incoming Shipment"
+                    class="oe_highlight"
+                    attrs="{'invisible':['|', '|', ('qty_to_receive', '=', 0), ('qty_to_receive', '&lt;', 0), ('state', '!=', 'approved')]}"
+                    type="action"
+                />
+                <button
+                    name="%(action_rma_picking_in)d"
+                    string="Create Incoming Shipment"
+                    attrs="{'invisible':['|', '|', ('qty_to_receive', '=', 0), ('qty_to_receive', '>', 0), ('state', '!=', 'approved')]}"
```

Comment:
This button would be hided if qty_to_receive = 0 but displayed if qty_to_receive < 0, it does not seem logic to me.
If it is visible when qty_to_receive < 0, it should also be visible when qty_to_receive = 0.
Also, I believe you need to hide this button if no lines have a receipt policy set, like it is on rma line side.

I believe we have 2 options : 
1) We behave in a different manner than on line side : we display only the receive button when we expected a reception (qty_to_receive > 0) and we don't display the button in the other case. Then, if the user want to force a reception, he should go on rma line where the button is displayed even if we don't expect him to generate a reception.
It may be  a bit confusing for the user to have a different behavior but if seems better/safer to me since on group, we manage multiple lines...

2) Act like on rma line side exactly, then you have to add a field on group with the receipt policy, to hide the button if no policy is set.

@sebastienbeau, it would be nice to have your opinion on this

Reply:
I think the best solution is the 1. User need a simple UI for processing the 99% of case. So only making visible the right button when there is something to do is the best option.
Then for the advanced case the user can go to the line and make some specific action.

Thanks for your work

Reply:
I agree with @sebastienbeau in this case. Better to stick to the most common use case.